lnd.go

```diff
@@ -93,10 +93,22 @@ var (
 	}
 )
 
+// ListenerCfg is a wrapper around custom listeners that can be passed to lnd
+// when calling its main method.
+type ListenerCfg struct {
+	// WalletUnlocker can be set to the listener to use for the wallet
+	// unlocker. If nil a regular network listener will be created.
+	WalletUnlocker net.Listener
+
+	// RPCListener can be set to the listener to use for the RPC server. If
+	// nil a regular network listener will be created.
+	RPCListener net.Listener
```

Comment:
Do we have any example code w.r.t how these are set and initialized in a mobile setting?

Reply:
These are generated by `falafel` on a per-service basis, as seen here: https://github.com/lightningnetwork/lnd/pull/3282/commits/0fb59d1d845276dbb6fafa55b4d3aeeab3bae856#diff-d258038f472562e6320fe2318a48ec8fR19